maven-core/src/test/java/org/apache/maven/repository/TestRepositorySystem.java

```diff
@@ -229,7 +229,7 @@ public ArtifactResolutionResult resolve(ArtifactResolutionRequest request) {
                             .map(Dependency::new)
                             .collect(Collectors.toList());
                 } catch (IOException e) {
-                    e.printStackTrace();
```

Comment:
Given that this is test code, Debug logging seems too weak. If this is a problem worth paying attention to, or that night explain why a test has failed or flaked, then bubble the IOException to immediately fail the test  with the root cause rather than waiting for it to fail down the line. 

On the other hand, if the test shouldn't and doesn't fail when this exception is thrown, then no debug log is needed.